src/Output.ts

```diff
@@ -25,10 +39,7 @@ export class Output {
     this._outputChannel.dispose();
   }
 
-  private static _outputChannel: IAzExtOutputChannel = createAzExtOutputChannel(
-    Constants.outputChannel.truffleForVSCode,
-    ext.prefix
```

Comment:
The `prefix` was actually not being used. It flows here https://github.com/microsoft/vscode-azuretools/blob/main/utils/src/AzExtOutputChannel.ts#L34-L38, where the timestamp was never logged.